common-app-lib/src/cmr/common_app/services/search.clj

```diff
@@ -1,15 +1,32 @@
 (ns cmr.common-app.services.search
   "This contains common code for implementing search capabilities in a CMR application"
   (:require [cmr.common.util :as u]
```

Comment:
Move namespaces in `require` to separate lines to match the latest naming convention. Pro Tip: this will increase your sloc count. :)